variants/heltec_wireless_stick_lite_v3/pins_arduino.h

```diff
@@ -0,0 +1,60 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define Wireless_Stick_Lite true
+#define DISPLAY_HEIGHT 0
+#define DISPLAY_WIDTH  0
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       15
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+static const uint8_t LED_BUILTIN = 35;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN LED_BUILTIN
+
+static const uint8_t TX = 44;
+static const uint8_t RX = 43;
+
+static const uint8_t SDA = 2;
+static const uint8_t SCL = 3;
+
+static const uint8_t SS    = 34;
+static const uint8_t MOSI  = 35;
+static const uint8_t SCK   = 36;
+static const uint8_t MISO  = 37;
+
+static const uint8_t A0 = 1;
+static const uint8_t A1 = 2;
+static const uint8_t A2 = 3;
+static const uint8_t A3 = 4;
+static const uint8_t A4 = 5;
+static const uint8_t A5 = 6;
+static const uint8_t A6 = 7;
+static const uint8_t A10 = 12;
+static const uint8_t A12 = 14;
+static const uint8_t A13 = 15;
+static const uint8_t A14 = 16;
+static const uint8_t A15 = 17;
+static const uint8_t A16 = 18;
+static const uint8_t A17 = 19;
+static const uint8_t A18 = 20;
+
+static const uint8_t T0 = 1;
+static const uint8_t T1 = 2;
+static const uint8_t T2 = 3;
+static const uint8_t T3 = 4;
+static const uint8_t T4 = 5;
+static const uint8_t T5 = 6;
+static const uint8_t T6 = 7;
```

Comment:
Should be touch `T1` to `T14` for the ESP32-S3. See page 19:
https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf

See also the Heltec Wifi Lora V3 pins:
https://github.com/espressif/arduino-esp32/blob/master/variants/heltec_wifi_lora_32_V3/pins_arduino.h

Reply:
I didn't placed them since they do not exist as an available pin in the board itself (https://resource.heltec.cn/download/Wireless_Stick_Lite_V3/HTIT-WSL_V3.png).

They can be placed if required?

Should I place them?

Reply:
That's a totally fair point. I added all of them because that's how it was [here](https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/blob/master/esp32/variants/WIFI_LoRa_32_V3/pins_arduino.h). I kind of agree with your point that we should only include pins which are actually available. But all the other heltec v3 pin files include all of them. Either way seems fine.

Reply:
I am hardly a tinkerer when it comes to electronics and for that reason I find it easier to think about what i can create/modify if there is a correlation between what I see in hardware and in code. 

That is why I believe showing only available pins is the right way to go.

If there is no issue I would like to be kept as is.

In the other it didn't make much sense my logic of separating them by ADC#_ and Ax and A1x so I did fix the numeration to be continuous.